docs/cluster_up_down.md

```diff
@@ -320,6 +321,26 @@ You can install logging aggregation components by specifying the --logging argum
 With logging aggregation installed, a new link will appear in the logs tab of a running pod in the web console.
 
 
+## Installing the Service Catalog
+
+| NOTE |
```

Comment:
i copied it from theo ther NOTEs in the file, they all look to be rendering fine?
https://github.com/bparees/origin/blob/6f84cb3a8422fc40eb46221300f7edd0241f4dfb/docs/cluster_up_down.md#installing-metrics